apps/site/app/globals.css

```
@@ -1,31 +1,15 @@
-@custom-variant dark (&:where(.dark, .dark *));
-
-:root {
-  --background: #FFFFFF;
-  --foreground: #1e293b;
-}
-
-.dark {
-  --background: #0a0a0a;
-  --foreground: #e2e8f0;
-}
+@source "../../../packages/ui/src/**/*.{ts,tsx}";
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)